src/khoj/routers/email.py

```diff
@@ -47,3 +46,38 @@ async def send_welcome_email(name, email):
             "html": html_content,
         }
     )
+
+
+from pydantic import BaseModel
+
+
+class TextData(BaseModel):
+    uquery: str
+    kquery: str
+    sentiment: str
+
+
+async def feedback(uquery, kquery, sentiment):
+    # console debug messages
+    logger.debug("SENDING USER FEEDBACK...")
+    logger.debug(f"User Query: {uquery}\n")
+    logger.debug(f"Khoj Response: {kquery}\n")
+    logger.debug(f"Sentiment: {sentiment}\n")
+    # rendering feedback email using feedback.html as template
+    template = env.get_template("feedback.html")
+    html_content = template.render(
+        uquery=uquery if not is_none_or_empty(uquery) else "N/A",
+        kquery=kquery if not is_none_or_empty(kquery) else "N/A",
+        sentiment=sentiment if not is_none_or_empty(sentiment) else "N/A",
+    )
```

Comment:
Can you also include the `user_email` in the template?

Reply:
Sure!